gravitee-apim-portal-webui/src/app/pages/application/application-creation/application-creation.component.ts

```diff
@@ -71,8 +71,8 @@ type ApplicationFormType = FormGroup<{
 
 function mapToApplicationInput(rawValue): ApplicationInput {
   const result = rawValue as ApplicationInput;
-  if (rawValue.oauth !== undefined) {
-    result.settings.oauth.additional_client_metadata = rawValue.oauth.additionalClientMetadata.reduce((acc, { key, value }) => {
+  if (rawValue.settings.oauth !== undefined) {
```

Comment:
```suggestion
  if (!!rawValue.settings.oauth) {
```
This means if `rawValue.settings.oauth` is truthy, rather than just not undefined.